lightpath/__main__.py

```diff
@@ -36,6 +36,10 @@ def main(db, hutches):
     db: str
         Path to happi JSON database
     """
+
+    from ophyd.signal import EpicsSignalBase
+    EpicsSignalBase.set_defaults(timeout=10.0, connection_timeout=10.0)
```

Comment:
This seems... fine. 

What are your thoughts on if we made this configurable as an environment variable-adjustable setting? `LIGHTPATH_EPICS_TIMEOUT` or perhaps even better, if done at the pcdsdevices level, `OPHYD_EPICS_TIMEOUT` or something like that

Reply:
It's a bit annoying since it seems to be only lightpath that runs into these issues at this point.  I'd say if we find any other cases where we want to increase the timeout, it might be worth making this into an environment variable. 

Otherwise I'm on board with pulling this into an environment variable, maybe we could set that in the conda environment?